README.md

```diff
@@ -1,2 +1,12 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+
+(a) The algorithm basically checks through the list to identify any duplications that may be present within the entries. If there are any, it will return as True. Then if not, it'll return as False.
+(b) If the list is in quadratic, it will need to run through the list either twice or multiple times depending on how many items are in the list. Adding more to the list will increase the amount of time taken. 
```

Comment:
Note that it's the algorithm that's quadratic, not the list. Basically for each element of the list the algorithm has to execute a linear search, hence n*n = n^2 operations.